.gitignore

```diff
@@ -171,6 +171,8 @@ tests/test_database/charleston/static/templates/overland/
 tests/test_database/charleston/input/measures/raise_property_aggregation_area/raise_property_aggregation_area_new.toml
 tests/system/sfincs/
 database
+tests/test_database/charleston/static/cyclone_track_database/
+tests/test_object_model/test_events.py
 tests/test_database/charleston/input/events/kingTideNov2021/gfs_anl_0p50_us_southeast.20211104_0000.nc
```

Comment:
can't you use wildcards instead of having each file? something like *.nc

Reply:
done, but my test database seems screwed up. None of the FIAT or benefits tests are working for me.